packages/service-clients/end-to-end-tests/azure-client/package.json

```diff
@@ -60,13 +61,15 @@
 		"@fluid-internal/mocha-test-setup": "workspace:~",
 		"@fluidframework/aqueduct": "workspace:~",
 		"@fluidframework/azure-client": "workspace:~",
+		"@fluidframework/azure-client-legacy": "npm:@fluidframework/azure-client@^1.2.0",
 		"@fluidframework/container-definitions": "workspace:~",
 		"@fluidframework/container-loader": "workspace:~",
 		"@fluidframework/core-interfaces": "workspace:~",
 		"@fluidframework/counter": "workspace:~",
 		"@fluidframework/datastore-definitions": "workspace:~",
 		"@fluidframework/fluid-static": "workspace:~",
 		"@fluidframework/map": "workspace:~",
+		"@fluidframework/map-legacy": "npm:@fluidframework/map@^1.4.0",
```

Comment:
IMO this is prob fine, at least for now.  Theoretically this means we need to be diligent about updating the import as we patch 1.x, but I'd defer engineering a solution unless/until we determine that's a frequent enough occurrence that it needs something automated.